src/Templates/src/templates/maui-mobile/Platforms/Windows/Package.appxmanifest

```diff
@@ -2,10 +2,13 @@
 <Package
   xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
   xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
   xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
   IgnorableNamespaces="uap rescap">
 
-  <Identity Name="maui-package-name-placeholder" Publisher="CN=User Name" Version="0.0.0.0" />
+  <Identity Name="maui-package-name-placeholder" Publisher="CN=User Name" Version="1.0.0.0" />
```

Comment:
The version should be 0.0.0.0 and the build tasks will replace it with the value in the csproj. If that is not working, then that is a separate bug that needs to be fixed before we release a servicing release.

Reply:
Leaving as 0.0.0.0